src/Resources/config/services.yml

```diff
@@ -92,6 +92,17 @@ services:
         class: Contao\CoreBundle\Security\User\ContaoUserProvider
         public: false
 
+    contao.store.session:
```

Comment:
True, but maybe we rename it to something more general anyway if it should also load the session on kernel.request again?